src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultTagHelperFactsService.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
In my most recent push, we don't call into this service anymore, but I had already nullable-annotated the file so decided to keep the changes.